moveit_simple/src/robot.cpp

```diff
@@ -377,8 +381,32 @@ std::unique_ptr<TrajectoryPoint> Robot::lookupTrajectoryPoint(const std::string&
   }
   else
   {
-    ROS_ERROR_STREAM("Failed to find point " << name << ", consider implementing more look ups");
-    throw std::invalid_argument("Failed to find point: " + name);
+    ROS_INFO_STREAM("Requesting trajectory training service info for waypoint: " << name);
+
+    std::vector<std::string> waypoint_names;
+    waypoint_names.push_back(name);
```

Comment:
Is this an overload?  If we send a trajectory with a single waypoint, it only returns that waypoint?  Should we create another service?

Reply:
GET_WP can return 1..n waypoints. It seemed unnecessary to create an additional case to ensure you only get a single waypoint. This call exists in the lookupTrajPoint call, but a lookupTrajectory could get a vector of waypoints. Same with lookupTrajPoints

Reply:
A service should have an obvious sematic meaning.  Getting a waypoint and getting a trajectory (a set of waypoints) is obvious.  I suppose getting multiple waypoints is also a use case, but it might be confused with getting a trajectory.  I guess I could be convinced that we have services for all 3 use cases, but we should be careful about trying to combine all use cases into a single service.  It's not like services cost money.